### DISCUSSION TITLE
Please help

### DISCUSSION THREAD

**Discussion by XahidEx** [vercel/community#976] [Help]
![Screenshot_20221022-132946_Firefox](https://user-images.githubusercontent.com/88249321/197326557-73bb8d91-b34b-4279-a85e-da56ff44a409.png)
Don't understand what's the problem. Deployment gets stuck without showing any errors. 
It used to take 2-3 minutes. Now It's not even building. 
[Deployment URL](https://xahidex-6xrs7nuz1-xahidex.vercel.app/)

**Comment by amyegan** [comment]
Sorry you're running into an error @XahidEx 😞 If you try to [redeploy](https://vercel.com/docs/concepts/deployments/managing-deployments#redeploy-a-project) your last working build, does it also run into the time limit now? Are you able to share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) or more details about what changed between the shorter builds and the longer one?
